src/main/java/com/configcat/Config.java

```diff
@@ -5,16 +5,36 @@
 import java.util.HashMap;
 import java.util.Map;
 
-class Config {
+/**
+ * Details of a ConfigCat config.
+ */
+public class Config {
+
+    /**
+     * The config preferences.
+     */
```

Comment:
I'd rather move these docs comments from private fields to the public accessors.